app/src/main/res/layout/fragment_supervision_response.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Not sure, but I think you can remove the `RelativeLayout`  here, as all it does is wrap the `LinearLayout`.

If I'm wrong though, you can ignore this comment, or explain that it is the case.

Reply:
Keep the RelativeLayout as it is holding multiple LinearLayout and Views within it.